docker/docker-compose.yml

```diff
@@ -75,20 +75,36 @@ services:
           - DEBUG
         volumes:
             - shared:/rpc
-            - lnd_data:/root/.lnd
         entrypoint: ["./start-lnd.sh"]
 
     lnd_ltc:
       extends: lnd
       container_name: lnd_ltc
       links:
           - "ltcd:blockchain"
+      volumes:
+          - lnd_ltc_data:/root/.lnd
 
     lnd_btc:
       extends: lnd
       container_name: lnd_btc
       links:
           - "btcd:blockchain"
+      volumes:
+          - lnd_btc_data:/root/.lnd
+
+    alice:
```

Comment:
Could you make the changes to this file a separate commit that explains what they achieve?